docs/geedocs/answer/2987876.html

```diff
@@ -22,7 +21,7 @@
 All rights reserved.</p>
 <p>Google, Google Earth, and Keyhole are trademarks or registered trademarks of Google, Inc. in the United States and elsewhere. Other trademarks not owned by Google, Inc. are the property of their respective owners.</p>
 <p>All Google and Keyhole logos and graphics are copyrighted. The content of this publication is furnished for informational use only, is subject to change without notice, and should not be construed as a commitment by Google, Inc.</p>
-<p>This publication is provided pursuant to and subject to the terms of the Google Earth Enterprise <gkms:snippet contenteditable="false" id="2995331"></gkms:snippet> and <gkms:snippet contenteditable="false" id="2995333"></gkms:snippet>. This publication and the information herein may not be reverse engineered, reproduced, duplicated, modified, altered, disclosed, published, printed, or distributed to any third party whatsoever without the express written permission of Google, Inc.</p>
+<p>This publication is provided pursuant to and subject to the terms of Google Earth Enterprise. This publication and the information herein may not be reverse engineered, reproduced, duplicated, modified, altered, disclosed, published, printed, or distributed to any third party whatsoever without the express written permission of Google, Inc.</p>
```

Comment:
I think this entire page needs to be overhauled, so I just wrote #271 to cover it.